testing/basic_server_tests.py

```diff
@@ -0,0 +1,18 @@
+"""
+basic_tests.py - Using testinfra to run tests on ansible playbook rke2-ansible
+"""
+
+import testinfra
+
+
+def test_rke2_config(host):
+    rke2_config = host.file("/etc/rancher/rke2/config.yaml")
+    assert rke2_config.user == "root"
+    assert rke2_config.group == "root"
+    assert rke2_config.mode == 0o640
```

Comment:
Same thing as basic_agent_tests.py, is 0o640 what you intended?

Reply:
This works as intended.